livre/core/data/NodeId.cpp

```diff
@@ -30,30 +30,26 @@ NodeId::NodeId( const Identifier& identifier )
     : _id( identifier )
 {}
 
-NodeId::NodeId( const uint32_t level,
-                const Vector3ui& position,
+NodeId::NodeId( const uint32_t level, const Vector3ui& position,
```

Comment:
I prefer the other way around. It is consistent in code ;) When it comes to style can we use each others ?